tests/lib/rules/complexity.js

```diff
@@ -124,7 +124,25 @@ ruleTester.run("complexity", rule, {
         { code: "class C { static { if (a || b) c = d || e; } }", options: [4], languageOptions: { ecmaVersion: 2022 } },
 
         // object property options
-        { code: "function b(x) {}", options: [{ max: 1 }] }
+        { code: "function b(x) {}", options: [{ max: 1 }] },
+
+        // optional chaining
+        {
+            code: "function a(b) { b?.c; }", options: [{ max: 2 }]
+        },
```

Comment:
What about optional call expressions like `function a(b) { b?.(); }`? Those should also increase the complexity I think.

Reply:
Good catch! I’ve added test cases and implemented handling of optional call expression.